charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -410,14 +410,35 @@ manager:
       repositoryDomain: public.ecr.aws/aws-observability
       repository: adot-autoinstrumentation-java
       tag: v1.32.3
+      resources:
```

Comment:
These are not related to the image.

Reply:
To assign different resource requests and limits per image, is that okay?

Reply:
Being addressed in https://github.com/aws-observability/helm-charts/pull/80.